mars/services/task/execution/ray/executor.py

```diff
@@ -67,6 +67,24 @@
 logger = logging.getLogger(__name__)
 
 
+# Metrics
+submitted_subtask_number = Metrics.counter(
+    "mars.band.submitted_subtask_number",
+    "The number of submitted subtask.",
+    ("session_id", "task_id", "stage_id"),
+)
+started_subtask_number = Metrics.counter(
+    "mars.band.started_subtask_number",
+    "The number of started subtask.",
+    ("subtask_id",),
+)
+completed_subtask_number = Metrics.counter(
+    "mars.band.completed_subtask_number",
+    "The number of completed subtask.",
+    ("subtask_id",),
```

Comment:
Maybe we should add tags `task_id`, `stage_id` for `started_subtask_number` and `completed_subtask_number` so that we can easily calculate a stage, a task total number of these two metrics.
If not, we can also calculate these number by specifying the time range.

Reply:
> Maybe we should add tags `task_id`, `stage_id` for `started_subtask_number` and `completed_subtask_number` so that we can easily calculate a stage, a task total number of these two metrics. If not, we can also calculate these number by specifying the time range.

Currently, the `execute_subtask` does not contains these info. I have to pass the `task_id` and `stage_id` to every Ray task for the tags.